lib/display/display.ts

```diff
@@ -7,6 +7,7 @@ import {
   Options,
   ScanResult,
   TestResult,
+  UnmanagedDependenciesMetaData,
```

Comment:
Let's sync on `UnmanagedDependenciesMetaData` tomorrow

Reply:
We synced 👍